index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Tumaini-Maganiko portfolio</title>
+</head>
+<body>
+    <header class="toolbar">
+        <div id="my-logo">
+            <span>My Logo</span>
+        </div>
```

Comment:
Since the logo is always a link that leads to the homepage, kindly consider putting it in a link tag (a tag). This is good for semantic reasons and provides better user experience. :wink: